content/docs/_index.md.bak

```diff
@@ -0,0 +1,38 @@
+---
```

Comment:
oops, you committed this backup file. Please `git rm` it and push another commit.